<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Community software tools for NGS by NCBI-Hackathons</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Community software tools for NGS</h1>
      <h2 class="project-tagline"></h2>
      <a href="https://github.com/NCBI-Hackathons/Community_Software_Tools_for_NGS" class="btn">View on GitHub</a>
      <a href="https://github.com/NCBI-Hackathons/Community_Software_Tools_for_NGS/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/NCBI-Hackathons/Community_Software_Tools_for_NGS/tarball/master" class="btn">Download .tar.gz</a>
    </section>

    <section class="main-content">
      <h3>
<a id="public-resources-related-to-clinical-impacts-of-snps" class="anchor" href="#public-resources-related-to-clinical-impacts-of-snps" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><a href="https://hermesfj.github.io/SNPs/" title="Metatools">Public Resources Related to Clinical Impacts of SNPs</a>
</h3>

<h3>
<a id="metagenomic-tools" class="anchor" href="#metagenomic-tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>[Metagenomic Tools]</h3>

<h3>
<a id="structural-variant-tools" class="anchor" href="#structural-variant-tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>[Structural Variant Tools]</h3>

<h3>
<a id="external-tools-that-work-wit-clinvar" class="anchor" href="#external-tools-that-work-wit-clinvar" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>[External Tools that work wit ClinVar]</h3>

<h3>
<a id="community-software-tools-for-ngs-list-of-community-software-tools-for-next-generation-sequencing" class="anchor" href="#community-software-tools-for-ngs-list-of-community-software-tools-for-next-generation-sequencing" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>[Community Software Tools for NGS](# List of Community Software Tools for Next Generation Sequencing</h3>

<h2>
<a id="commercial-tools" class="anchor" href="#commercial-tools" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Commercial tools</h2>

<ol>
<li>
<strong><a href="http://www.strand-ngs.com/">Strand NGS</a></strong>

<ul>
<li>offers many different tools including alignment, RNA-Seq, DNA-Seq, ChIP-Seq, Small RNA-Seq, Genome Browser, visualizations, Biological Interpretation, etc. Supports workflows
“one can import the sample data in FASTA, FASTQ or tag-count format. In addition, prealigned data in SAM, BAM or Illumina-specific ELAND format can be directly imported for analysis.”</li>
<li>Alignment feature: Supports alignment from Illumina, Ion Torrent, 454 (Roche), and Pac Bio</li>
<li>DNA-Seq Feature, can annotate with dbSNP</li>
</ul>
</li>
<li>
<strong><a href="http://www.clcbio.com/desktop-applications/top-features/">CLC Genomics Workbench</a></strong><br>

<ul>
<li>(QIAGEN). Features include: resequencing, workflow, read mapping, de novo assembly, variant detection, RNA-Seq, ChIP-Seq, Genome Browser, etc (entire list on website); Main Workbench offers database search (Genbank, Blast, Pubmed); 2000 organizations have invested in CLC</li>
<li>Accepts VCF files from 1000 Genomes Project</li>
<li>Accepts downloaded tracks from dbSNP</li>
<li>Also accepts: FASTA, GFF/GTF/GVF, BED, Wiggle, Cosmic, UCSC variant database, complete genomics master var file</li>
<li>Read mapping: “In addition to Sanger sequence data, reads from these high-throughput sequencing machines are supported: The 454 FLX System and the 454 GS Junior System from Roche, Illumina Genome Analyzer, Illumina HiSeq, Illumina HiScan, and Illumina MiSeq sequencing systems, SOLiD system from Life Technologies, Ion Torrent system from Life Technologies, Helicos from Helicos BioSciences”</li>
<li>De novo assembly: “In addition to Sanger sequence data, reads from these high-throughput sequencing machines are supported The 454 FLX System and the 454 GS Junior System from Roche, Illumina Genome Analyzer, Illumina HiSeq, Illumina HiScan, and Illumina MiSeq sequencing systems, SOLiD system from Life Technologies, Ion Torrent system from Life Technologies”</li>
<li>Annotation tracks from Ensembl</li>
</ul>
</li>
<li>
<strong><a href="https://www.dnanexus.com/product-overview">DNAnexus</a></strong>

<ul>
<li>Private cloud repository -- formerly a redistributor of SRA and other NCBI resources; command-line or via web, can fetch data from a URL,  build custom pipeline/ workflow
has sra.dnanexus.com site: data downloads come directly from NCBI</li>
</ul>
</li>
<li>
<strong><a href="http://www.ingenuity.com/products/variant-analysis">Ingenuity Variant Analysis</a></strong>

<ul>
<li>(QIAGEN) allows for variant identification and analysis, uses NCI-60 data set for cancer, Supported third part informatin: Entrez Gene, RefSeq, ClinVar; gives contextual details of results instead of just A to B relationship</li>
<li>Has own database-- “knowledge base” based on COSMIC, OMIM, and TCGA databases</li>
</ul>
</li>
<li>
<strong><a href="http://www.dnastar.com/t-products-dnastar-lasergene-genomics.aspx">Lasergene Genomics Suite</a></strong>

<ul>
<li>Comprehensive NGS software pipeline for assembly, alignment, variant calling and analysis of NGS data</li>
<li>Supported workflows include: reference-guided and de novo genome and transcriptome assembly and analysis, metagenomics sample assembly, targeted resequencing, exome alignment, gene panels with validation control, variant analysis, and RNA-Seq, ChIP-Seq and miRNA alignment and analysis. </li>
<li>#1 in accuracy: fewer false negatives and better sensitivity compared to results obtained from other aligners</li>
<li>Aligns exome data and performs variant calling an average of 3 times faster than alternative pipelines</li>
<li>Annotates genomic data with allele and genotype frequency, functional impact predictions, evolutionary conservation scores and pathogenicity</li>
<li>Supports all major NGS technologies (Illumina, Ion Torrent, Pac Bio and Roche 454) and project types</li>
<li>Available on Windows, Mac OS X, Linux, and the Amazon Cloud</li>
</ul>
</li>
<li>
<strong><a href="http://www.softgenetics.com/NextGENe.html">NextGENe</a></strong>

<ul>
<li>“perfect analytical partner for the analysis of desktop sequencing data produced by the ION PGM™, Roche Junior, Illumina MiSeq as well as high throughput systems as the Ion Torrent Proton, Roche FLX, Applied BioSystems SOLiD™ and Illumina® platforms.” runs on Windows, free-standing multi-application package-- SNP/Indel analysis, CNV prediction and disease discovery, whole genome alignment, etc.<br>
</li>
<li>Data can be imported from Clinvar, dbSNP, Genbank: <a href="http://www.softgenetics.com/PDF/NextGene_UsersManual_web.pdf">http://www.softgenetics.com/PDF/NextGene_UsersManual_web.pdf</a>
</li>
</ul>
</li>
<li>
<strong><a href="http://www.partek.com/pgs">Partek Genomics Suite</a></strong>

<ul>
<li>Cited in over 3,500 peer-reviewed scientific publications</li>
<li>Workflows for microarray and PCR data include: Gene expression including alternative splicing, miRNA expression, Genome Wide Association Studies, Mother-Father-Child Trio analysis, DNA Copy number including allele specific copy number and Loss of Heterozygosity (LOH), and ChIP, and methylation. Next Generation Sequencing (NGS) workflows include: RNA-Seq, miRNA-Seq, ChIP-Seq, DNA-Seq, and Methylation</li>
<li>Powerful statistics and interactive, publication ready visualizations</li>
<li>Supports all commercial next generation sequencing and microarray file format as well as text files</li>
<li>Can input GEO SOFT files</li>
</ul>
</li>
<li>
<strong><a href="http://www.partek.com/partekflow">Partek Flow</a></strong>

<ul>
<li>Installation can be cloud-based or on a local cluster or Linux server</li>
<li>Easy to use point-and-click interface</li>
<li>Takes NGS data (.fastq, BAM, SAM), microarrays (Affymetrix, Illumina) and text files</li>
<li>Supports custom genome builds and annotation databases</li>
<li>Performs base trimming, alignment, quantification, quality analysis, statistics, and visualization</li>
<li>Includes ten fully customizable aligners (Bowtie, Bowtie 2, BWA, GSNAP, Isaac 2, SHRiMP 2, STAR, TMAP, TopHat and TopHat 2)</li>
<li>Applications for RNA-Seq, Small RNA-Seq, WGS/WES, Pathway enrichment, Fusion detection and Variant calling</li>
<li>Allows users to create, save, share, or download analysis pipelines for automated and repeatable analysis</li>
<li>Collaborate with others without transferring data</li>
<li>Integrates microarray and next generation sequencing data</li>
</ul>
</li>
<li>
<strong><a href="http://goldenhelix.com/SNP_Variation/">Golden Helix: SNP and Variation Suite</a></strong>

<ul>
<li>used for managing, analyzing and visualizing genotypic and phenotypic data; Features: Genome-wide association studies, genomic prediction, copy number analysis, small sample DNA-Seq workflows, large sample DNA-seq analysis, RNA-seq analysis.
Supported files: .txt, excel XLS &amp; XLSX, CEL, CHP, CNT, Illumina, Plink PED, TPED, BED, Agilent files, NimbleGen data summary files, VCF files, Impute2 GWAS files, HapMap format, MACH output, + 50 other formats
consumes NCBI data directly</li>
</ul>
</li>
<li>
<strong><a href="https://www.genomatix.de/">Genomatix</a></strong>

<ul>
<li>Applications: ChIP-Seq, DNA-Seq, RNA-Seq, DNA methylation; enable personalized medicine,</li>
<li>Mining Stations: Supports all established NGS sequencing platforms- SOLiD, 454 Life Sciences, Genome Analyzer, HiSeq, MiSeq, IonTorrent</li>
<li>Software Suite: can upload sequence of BED files</li>
<li>Genome browser: BED and BAM files, Public data- 1500 BED files available for every user</li>
</ul>
</li>
<li>
<strong><a href="http://www.biodatomics.com/">Biodatomics</a></strong>

<ul>
<li>Open source platform (SaaS), analysis and genome sequencing tools, integrates over 400 genomic analysis open source tools and pipelines, have a private and public cloud version. Features: genomic data visualization, drag and drop interface, accelerated analysis, real-time collaboration</li>
<li>They have a couple modules to do so, and have enabled parts of the sra toolkit</li>
</ul>
</li>
<li>
<strong><a href="https://www.solvebio.com/">SolveBio</a></strong>

<ul>
<li>Software product, for clinical genomics professionals, manage, curate, report genomic variation</li>
<li>Has own data library -- data from NCBI</li>
</ul>
</li>
</ol>

<h2>
<a id="variant-identification" class="anchor" href="#variant-identification" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Variant Identification</h2>

<h3>
<a id="germline-callers" class="anchor" href="#germline-callers" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Germline Callers</h3>

<ol>
<li>
<strong><a href="http://mathgen.stats.ox.ac.uk/impute/impute_v2.html">IMPUTE2</a></strong>

<ul>
<li>Description: phasing observed genotypes and imputing missing genotypes uses reference panels to provide all available halotypes, does not use population labels or genome-wide measures; designed to represent variation in one population; Fairly popular</li>
<li>Input: </li>
<li>Reference Haplotypes: Links to 1000 Genomes and HapMap downloads</li>
<li>Output: </li>
</ul>
</li>
<li>
<strong><a href="https://github.com/ekg/freebayes">FreeBayes</a></strong>

<ul>
<li>Description: finds SNPs, Indels, MNPs; reports variants based on alignment; haplotype based</li>
<li>Input: BAM- uses BAMtools API to parse</li>
<li>Reference genome: FASTA</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://soap.genomics.org.cn/soapindel.html">SOAPindel</a></strong>

<ul>
<li>Description: detects indels from NGS paired-end sequencing</li>
<li>Input: files with read alignment can be SOAP or SAM formats, users must also give raw reads in Fasta or Fastq</li>
<li>Reference Sequence used to align reads: FASTA</li>
<li>Output:
<hr>
</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/danmaclean/2kplus2">2Kplus2</a></strong>

<ul>
<li>Description: algorithm searches graphs produced by de novo assembler Cortex; c++ source code for SNP detection
“2kplus2.cpp is a c++ source code for the detection and the classification of single nucleotide polymorphisms in transformed De Bruijn graphs using Cortex assembler.”</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="https://www.hgsc.bcm.edu/software/atlas-2">Atlas 2</a></strong>

<ul>
<li>Description: specializes in separation of true SNPs and indels from sequencing and mapping errors, last update January 2013</li>
<li>Input: takes BAM file,</li>
<li>Reference Genome: FASTA</li>
<li>Output: produces VCF</li>
</ul>
</li>
<li>
<strong><a href="https://sites.google.com/site/vibansal/software/crisp">CRISP</a></strong>

<ul>
<li>Description: identifies SNPs and INDELs from pooled high-throughput NGS, not used for analysis of single samples; implemented in C and uses SAMtools API; latest version should work with diploid genomes</li>
<li>Input: requires BAM files (aligned with GATK)</li>
<li>Reference Genome: indexed FASTA file</li>
<li>Output: VCF files</li>
</ul>
</li>
<li>
<strong><a href="http://www.sanger.ac.uk/resources/software/dindel/">Dindel</a></strong>

<ul>
<li>Description: (Wellcome Trust Sanger) calls small indels from short-read sequences, only can handle Illumina data; cannot test candidate indels; written in C++, used on Linux based and Mac computers (not tested in windows)</li>
<li>Input: BAM files</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://colibread.inria.fr/software/discosnp/">discoSnp++</a></strong>

<ul>
<li>Description: detects homozygous and heterozygous SNPs and Indels; software composed of 2 modules (kissnp2 and kissreads)</li>
<li>Input: raw NGS datasets; fasta, fastq, gzipped or not;</li>
<li>no reference genome required; read pairs can be given</li>
<li>Output: FASTA</li>
</ul>
</li>
<li>
<strong><a href="http://odin.mdacc.tmc.edu/%7Ewwang7/FamSeqIndex.html">FamSeq</a></strong>

<ul>
<li>Description: family-based sequencing studies- provides probability of an individual carrying variant based on family’s raw measurements; accommodates de novo mutations, can perform variant calling at chrX;</li>
<li>Input: VCF</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/p/%20geneticthesaurus/wiki/Example/">GeneticThesaurus</a></strong>

<ul>
<li>Description: “Annotation of genetic variants in repetitive regions”</li>
<li>Input: Initial variant calling from bam → vcf output</li>
<li>Reference Genome: need to provide own fasta file for hg19 genome,<br>
</li>
<li>Output: vcf.gz, vtf.gz, and baf.tsv.gz output</li>
</ul>
</li>
<li>
<strong><a href="http://genome.sph.umich.edu/wiki/GlfMultiples">glfMultiples</a></strong>

<ul>
<li>Description: command-line, variant caller</li>
<li>Input: GLF</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://genome.sph.umich.edu/wiki/GlfSingle">glfSingle</a></strong>

<ul>
<li>Description: uses likelihood-based model for variant calling, starts from genotype likelihoods that have been computed from other tools (ex. Samtools BAQ), the likelihoods combine with individual-based prior p(genotype) to generate posterior probabilities</li>
<li>Input: GLF</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/ddcap/halvade">Halvade</a></strong>

<ul>
<li>Description: command-line; written in Java, “to run halvade a reference is needed for both GATK and BWA and a SNP (dbSNP!) database is required</li>
<li>Input: FASTQ</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/aakrosh/indelMINER">indelMINER</a></strong>

<ul>
<li>Description: identifies indels from paired-end reads</li>
<li>Input: BAM (aligned in SAMtools API)</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="https://www.broadinstitute.org/cancer/cga/indelocator">Indelocator</a></strong>

<ul>
<li>Description: (Broad Institute): does not perform realignment, relies on alignments in BAM files (BAM files need aligned before put into indelocator); recommended to use GATK prior;</li>
<li>Input: 2 BAM files(tumor &amp; normal), annotated as germline or somatic; also has single sample mode</li>
<li>Output: “Output of Indelocator is a high-sensitivity list of putative indel events containing large numbers of false positives. The statistics reported for each event have to be used to custom-filter the list in order to lower false positive rate”</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/sequencing/isaac_variant_caller">Isaac Variant Caller</a></strong>

<ul>
<li>Description: detects SNPs and small indels from diploid sample; designed to run on “nux-like platforms”</li>
<li>Input: BAM</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://www.swisstph.ch/kvarq">KvarQ</a></strong>

<ul>
<li>Description: in silico genotyping for selected loci in bacterial genome, written in Python and C</li>
<li>Input: FASTQ</li>
<li>reference genome or de novo assembly not needed</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/projects/lofreq/files/">LoFreq</a></strong>

<ul>
<li>Description: SNV caller, Python language, standalone program, uncovers cell-population heterogeneity from high-throughput sequencing datasets; calls variants found in &lt;.05% of the population</li>
<li>Input: BAM file input→ suggest running through GATK</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/Illumina/manta">Manta</a></strong>

<ul>
<li>Description: Calls indels and SVs from paired end reads; standalone, command line program; Written in C++ and Python</li>
<li>Input: BAM (can tolerate non-paired-end reads); a matched tumor sample may be provided as well</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/benedictpaten/marginAlign">MarginAlign</a></strong>

<ul>
<li>Description: SNV caller, specifically tailored to Oxford Nanopore Reads, written in Python; Package comes with 3 programs, marginAlign, marginCaller (calls SNVs), marginStats (computes qc stats on sam files)</li>
<li>Input: SAM</li>
<li>Output: SAM</li>
</ul>
</li>
<li>
<strong><a href="http://gmt.genome.wustl.edu/packages/mendelscan/">MendelScan</a></strong>

<ul>
<li>Description: Last release March 2014; for analyzing sequencing data in family studies of inherited diseases; variant calls for a family in VCF file; still in alpha-testing
on github, example data uses 1000 genomes dataset</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/mitenjain/nanopore">nanopore</a></strong>

<ul>
<li>Description: UCSC Nanopore group (group at UCSC studying using ion channels for analysis of single RNA/DNA structures) software pipeline; tailored to Oxford Nanopore Reads; command line program</li>
<li>Input: FASTQ</li>
<li>Reference files: FASTA</li>
<li>Output: “For each possible pair of read file, reference genome and mapping algorithm an experiment directory will be created in the nanopore/output directory.”</li>
</ul>
</li>
<li>
<strong><a href="http://omictools.com/platypus-s1989.html">Platypus</a></strong>

<ul>
<li>Description: Package program, written in C, Python, Cython;  Can identify SNPs, MNPs, short indels, and larger variants; has been tested on very large datasets (1000 genomes)</li>
<li>Input: BAM</li>
<li>Reference Genome: FASTA (files must be indexed using Samtools or similar program</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://www.bioinformatics.nl/QualitySNPng/">QualitySNPng</a></strong>

<ul>
<li>Description: detection of SNPs; “can be used as a standalone application with graphical user interface as part of pipeline system”; does not require fully sequenced reference genome; haplotype strategy</li>
<li>Input:SAM, ACE</li>
<li>Output: GUI</li>
</ul>
</li>
<li>
<strong><a href="http://revister.sourceforge.net/">ReviSTER</a></strong>

<ul>
<li>Description: command line program; automated pipeline; utilizes BWA, BLAT, and SAMTools;  utilizes BWA mapping program;</li>
<li>Input: FASTQ,</li>
<li>Reference sequence file and list file containing STR locations as inputs</li>
<li>Output: SAM</li>
</ul>
</li>
<li>
<strong><a href="http://dna-discovery.stanford.edu/software/rvd/">RVD</a></strong>

<ul>
<li>Description: command-line program, detection of rare SNVs, relies upon Samtools, can be run in MATLAB</li>
<li>Input: BAM</li>
<li>Reference Genome: FASTA</li>
<li>Output: “The algorithm output is a call table -- a comma-separated file with one line for each base position and each line in the following format:</li>
<li>AlginmentReferencePosition, AlignmentBase, Call ,SecondBase, CenteredErrorPrc, ReferenceErrorPrc, SecondBasePrc”</li>
</ul>
</li>
<li>
<strong><a href="http://snver.sourceforge.net/">SNVer</a></strong>

<ul>
<li>Description: calls common and rare variants in pool or individual NGS data, reports overall p-value, operating system independent statistical tool, identifies SNPs and INDELs, written in Java, no dependencies, straightforward command-line<br>
</li>
<li>(SNVerGUI=GUI version) --SNVerGUI: desktop tool for variant detection</li>
<li>Input: chrX annotation, sam.zip, bam.zip</li>
<li>reference file must be aligned to the data file</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://compbio.bccrc.ca/software/snvmix/">SNVMix</a></strong>

<ul>
<li>Description: detects SNVs from NGS, post-alignment tool</li>
<li>Input: pileupformat (Maq or Samtools)</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://www.bsse.ethz.ch/mlcb/research/bioinformatics-and-computational-biology/structural-variant-machine--sv-m-.html">SV-M</a></strong>

<ul>
<li>Description: Structural Variant Machine - predicts indels, uses split read alignment profiles, validated by Sanger Sequencng</li>
<li>Input:paired-end Illumina reads from 1001 genomes project (uses ref plant- 1001genomes.org)</li>
<li>Ouptut:</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/slindgreen/SNPest">SNPest</a></strong>

<ul>
<li>Description: Standalone program, language C++, Perl</li>
<li>Input: mpileup (SAMtools)</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://genome.sph.umich.edu/wiki/TrioCaller">TrioCaller</a></strong>

<ul>
<li>Description:Command line program, relies on BWA and samtools; genotype calling for unrelated individuals and parent-offspring trios</li>
<li>Input: BAM (that has been aligned in BWA and Samtools</li>
<li>Output: BCF that can be formatted to VCF using bcftools</li>
</ul>
</li>
<li>
<strong><a href="http://www.vicbioinformatics.com/software.snippy.shtml">Snippy</a></strong>

<ul>
<li>Description: finds indels between haploid reference genome and NGS sequence reads</li>
<li>Input:read files- FASTQ or FASTA (can be .gz compressed), output- .aln, .tab, .txt</li>
<li>Reference genome in FASTA or GENBANK</li>
<li>Output: </li>
</ul>
</li>
<li>
<strong><a href="http://orca.bu.edu/vntrseek/">VntrSeek</a></strong>

<ul>
<li>Description: pipeline for discovering microsatellite tandem repeats with high-throughput sequencing data</li>
<li>Input: gzip-compressed FASTA or FASTQ</li>
<li>Output: VCF files; one for TRs and observed alleles, another file contains link to viewer</li>
</ul>
</li>
</ol>

<h3>
<a id="somatic-callers" class="anchor" href="#somatic-callers" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Somatic Callers</h3>

<ol>
<li>
<strong><a href="http://cakesomatic.sourceforge.net/">Cake</a></strong>

<ul>
<li>Description: standalone program, “pipeline for the integrated analysis of somatic variants in cancer genomes”; integrates four algorithms; written in Perl; required tools: samtools, tabix, vcftools, VarScan2, bambino, cmake, somaticsniper (User guide; workflow page)</li>
<li>Input: tumor and normal reads in BAM files, run through variant calling programs to generate intermediate VCF</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://www.broadinstitute.org/cancer/cga/mutect">MuTect</a></strong>

<ul>
<li>Description: Broad Institute, identification of somatic point mutations in cancer genomes; requires preprocessing of reads (GATK)</li>
<li>Input: same as GATK (FASTA reference genome, SAM read files)</li>
<li>Output: call-stats, VCF, wiggle files</li>
</ul>
</li>
<li>
<strong><a href="http://genome.sph.umich.edu/wiki/Polymutt">Polymutt</a></strong>

<ul>
<li>Description: calls SNVs and detects de novo point mutations in families</li>
<li>Input: GLF or BAM or VCF (must have identical chromosome orders)</li>
<li>Output: VCF
<hr>
</li>
</ul>
</li>
<li>
<strong><a href="http://tvap.genome.wustl.edu/tools/bassovac/">Bassovac</a></strong>

<ul>
<li>Description: Improved Bayesian inversion somatic caller; unlike other software packages, treats effects fully probabilisticallys instead of using ad-hoc modeling; effects are integrated at the atomic level and standard probability theory integrates read tallies to the sample level and to the tumor-normal pair level; "pending public release"</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://bioinformatics.ustc.edu.cn/CLImAT/">CLImAT</a></strong>

<ul>
<li>Description: standalone program; “accurate detection of copy number alteration and loss of heterozygosity in impure and aneuploid tumor samples using whole genome sequencing data”</li>
<li>Input: depth file generated by DFExtract and a config file</li>
<li>Output: .results file, .Gtype, LOG.txt, also generates visualization</li>
</ul>
</li>
<li>
<strong><a href="http://denovogear.sourceforge.net/">DeNovoGear</a></strong>

<ul>
<li>Description: de-novo variant calling and interpretation; standalone program; dependencies C++ compiler, CMake, HTSlib, Eigen, Boost</li>
<li>Input: PED and BCF</li>
<li>Output: “The output format is a single row for each putative de novo mutation (DNM), with the following fields”</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/friend1ws/EBCall">EBCall</a></strong>

<ul>
<li>Description: Empirical Baysian Mutation Calling; standalone program; uses tumor/normal paired reads and non-paired normal reference samples; dependent on samtools, R and VGAM pack for R</li>
<li>Input: BAM</li>
<li>Output: not sure what exact type of file- “The format of the result is suitable for adding annotation by annovar.”</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/usuyama/hapmuc">HapMuc</a></strong>

<ul>
<li>Description: standalone program; “utilizes the information of heterozygous germline variants near candidate mutations”; Dependent upon- Boost, SAMtools, BEDtools; 3 step workflow</li>
<li>Input: BAM</li>
<li>Output: BED</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/cui-lab/multigems">MultiGeMS</a></strong>

<ul>
<li>Description: Multi-sample Genotype Model Selection</li>
<li>Input: .txt, pileup (SAM/BAM converted to pileup format)</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="https://bitbucket.org/joseph07/multisnv/wiki/Home">MultiSNV</a></strong>

<ul>
<li>Description: command-line program; calls SNVs from NGS data from multiple samples from the same patient; dependent on R, Git, cmake, Boost and compile libraries</li>
<li>Input: BAM or pileup</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://compbio.bccrc.ca/software/mutationseq/">MutationSeq</a></strong>

<ul>
<li>Description: standalone program, somatic SNV detection in tumor/normal samples; dependent on python, bamtools, boost, and LAPACK</li>
<li>Input: BAM</li>
<li>Output: VCF4.1 consisting of two parts (meta information &amp; data lines)</li>
</ul>
</li>
<li>
<strong><a href="http://www.qcmg.org/bioinformatics/tiki-index.php">qSNP</a></strong>

<ul>
<li>Description: standalone program; SNV caller for somatic variants in “low cellularity cancer samples”</li>
<li>Input: BAM, dbSNP data, Illumina data, chrConv</li>
<li>Output: “qSNP output files are named using a 4-element pattern: ...”</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/aradenbaugh/radia/">RADIA</a></strong>

<ul>
<li>Description: RNA and DNA Integrated Analysis for Somatic Mutation Detection; DNA only Method(tumor/normal pair, ignores RNA) or Triple BAM Method (uses all three datasets from same patient); dependent upon python, samtoools, pysam API, BLAT, SnpEff</li>
<li>Input: BAM</li>
<li>Reference Genome: FASTA indexed with SAMtools faidx</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://genomics.wpi.edu/rvd2/">RVD2</a></strong>

<ul>
<li>Description: sensitive, variant detection for low-depth targeted NGS data; python module or command- line program;</li>
<li>Input: tab- deliminted depth chart format (converted from pileup files)</li>
<li>Output: three hdf5 files and a vcf file</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/nhansen/Shimmer">Shimmer</a></strong>

<ul>
<li>Description: standalone program; detects somatic SNVs with multiple testing correction, uses Fisher’s exact test; dependent on git, samtools, R, R statmod package; for tumor/normal matched samples</li>
<li>Input: BAM</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://www.cs.helsinki.fi/en/gsa/snv-ppilp/">SNV-PPILP</a></strong>

<ul>
<li>Description: Refines GATK’s Unified Genotyper SNV calls for “multiple samples assumed to form a phylogeny”</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://gmt.genome.wustl.edu/packages/somatic-sniper/">SomaticSniper</a></strong>

<ul>
<li>Description: command-line application to identify SNPs between tumor/normal pairs- predicts probability of difference between two</li>
<li>Input: BAM</li>
<li>Reference Genome in FASTA</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="https://sites.google.com/site/strelkasomaticvariantcaller/">Strelka</a></strong>

<ul>
<li>Description: somatic variant calling workflow for matched tumor-normal samples; detects indels; runs on *nux-like platform</li>
<li>Input: BAM (must be sorted and indexed)- Strelka does own realignment around indels-- don’t need to do this type of pre-processing</li>
<li>Output: pair of VCF files</li>
</ul>
</li>
<li>
<strong><a href="http://www.pitt.edu/%7Ewec47/triodenovo.html">Triodenovo</a></strong>

<ul>
<li>Description: Bayesian framework for calling de novo mutations in trios</li>
<li>Input: VCF file with PL or GL fields (recommend using GATK or samtools to generate)</li>
<li>Output: out_vcf</li>
</ul>
</li>
<li>
<strong><a href="http://lbg.med.unc.edu/%7Emwilkers/unceqr_dist/">UNCeqr</a></strong>

<ul>
<li>Description: finds somatic mutations using integration of DNA and RNA seq data-- boosts sensitivity for low purity tumors and rare mutations;</li>
<li>Input:”can accept a variety of sequencing inputs and configurations”</li>
<li>Output: “table of somatically mutated sites and associated information. These somatic mutations can be annotated with predicted transcript and protein effects using third party tools, such as Annovar”</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/projects/virmid/">Virmid</a></strong>

<ul>
<li>Description: Virtual Microdissection for SNP calling; Java based; for disease-control matched samples; uncovers SNPs with low allele frequency by considering alpha contamination</li>
<li>Input: BAM (must be sorted and indexed- samtools sort)</li>
<li>Output: VCF and report file</li>
</ul>
</li>
</ol>

<h3>
<a id="germline--somatic--callers" class="anchor" href="#germline--somatic--callers" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Germline + Somatic  Callers</h3>

<ol>
<li>
<strong><a href="http://massgenomics.org/varscan">VarScan 2</a></strong>

<ul>
<li>Description: identify germline variants, private and shared variants, somatic mutations, and somatic CNVs; detects indels</li>
<li>Input: SAMtools pileup</li>
<li>Output: VCF
<hr>
</li>
</ul>
</li>
<li>
<strong><a href="http://genformatic.com/baysic/">BAYSIC</a></strong>

<ul>
<li>Description: Bayesian method; combines variant calls from different methods (GATK, FreeBayes, Atlas, Samtools, etc)</li>
<li>Input: VCF format from one or more variant calling programs</li>
<li>Output: VCF file containing integrated set of variant calls</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/ding-lab/msisensor">MSIsensor</a></strong>

<ul>
<li>Description: Microsatellite instability detection; C++ program, detects somatic and germline variants in tumor-normal paired data</li>
<li>Input: BAM index files (normal and tumor)</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://faculty.washington.edu/browning/beagle/beagle.html">Beagle version 4</a></strong>

<ul>
<li>Description: software package: genotype calling, phasing, imputation of ungenotyped markers, and identity-by-descent segment detection:unsure if this one is in the right category; genotype calling, phasing, imputation of ungenotyped markers, and identity-by-descent segment detection; </li>
<li>Input: VCF</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://www.iro.umontreal.ca/%7Ecsuros/quadgt/">QuadGT</a></strong>

<ul>
<li>Description: software package, SNV calling from normal-tumor pair and two parent genomes; quantifies descent-by-modification relationships; Written in Java</li>
<li>Input: BAM files (parsed by Picard/Samtools API)</li>
<li>Reference Genome; FASTA</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/projects/rarevator/">RAREVATOR</a></strong>

<ul>
<li>Description: RAre REference VAriant annotaTOR; command line;  “identification and annotation of germline and somatic variants in rare reference allele loci from second generation sequencing data”; Bayesian genotype likelihood model</li>
<li>Input: BED or VCF files from GATK</li>
<li>Output: two VCF files (one for SNVs, one for Indels)</li>
</ul>
</li>
<li>
<strong><a href="http://scalpel.sourceforge.net/">Scalpel</a></strong>

<ul>
<li>Description: Used for detecting indels in a reference genome; performs localized micro-assembly of specific regions of interest; can do single, de novo, somatic reads; requires that raw reads are aligned with BWA</li>
<li>Input: BAM</li>
<li>Output: either VCF or ANNOVAR</li>
</ul>
</li>
<li>
<strong><a href="http://soap.genomics.org.cn/soapsnp.html">SOAPsnp</a></strong>

<ul>
<li>Description: based on Baye’s theorem; calls consensus genotype</li>
<li>Input:SOAP short read alignment results</li>
<li>Output: GLF, option of flat tabular format</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/projects/variantmaster/">VariantMaster</a></strong>

<ul>
<li>Description: “extract causative variants for monogenic and sporadic genetic diseases”; uses ANNOVAR;</li>
<li>Input: BAM or VCF files (from SAMtools, GATK)</li>
<li>Output:</li>
</ul>
</li>
</ol>

<h2>
<a id="downstream-analysis-of-variants" class="anchor" href="#downstream-analysis-of-variants" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Downstream Analysis of Variants</h2>

<ol>
<li>
<strong><a href="https://github.com/hakyimlab/PrediXcan%20https://github.com/hriordan/PrediXcan/">PrediXcan</a></strong>

<ul>
<li>Description: command-line, standalone package program; available in Perl, Python, and R versions; predicts liklihood of a gene being related to a certain phenotype- “that directly tests the molecular mechanisms through which genetic variation affects phenotype.”; no actual expression data used, only in silico expression; “PrediXcan can detect known and novel genes associated with disease traits and provide insights into the mechanism of these associations.”</li>
<li>Input: genotype and phenotype file (doesn’t specify file type)</li>
<li>Output:default values: genelist, dosages (file format:  snpid rsid) , dosage_prefix, weights, output
<hr>
</li>
</ul>
</li>
<li>
<strong><a href="http://ritchielab.psu.edu/software/athena-downloads">ATHENA</a></strong>

<ul>
<li>Description: Analysis Tool for Heritable and Environmental Network Associations; software package, combines machine learning model with biology and statistics to predict non-linear interactions</li>
<li>Input: Configuration file, Data file, Map file (includes rsID)</li>
<li>Output: Summary file, Best model file, dot file, individual score file, cross-validation file</li>
</ul>
</li>
<li>
<strong><a href="http://www.sanger.ac.uk/resources/software/rarevariant/#t_2">CCRaVAT and QuTie</a></strong>

<ul>
<li>Description: (Wellcome Trust Sanger) Case-Control Rare Variant Analysis Tool and Quantitative Trait; software packages for large-scale analysis of rare variants</li>
<li>Input: PED file and MAP file</li>
<li>Output: Five tab-delimited txt files</li>
</ul>
</li>
<li>
<strong><a href="http://cnsgenomics.com/software/gcta/">GCTA</a></strong>

<ul>
<li>Description: Genome Wide Complex Trait Analysis; package program, command line interface; estimates variance by all SNPs; 5 main functions: “data management, estimation of the genetic relationships from SNPs, mixed linear model analysis of variance explained by the SNPs, estimation of the linkage disequilibrium structure, and GWAS simulation”</li>
<li>Input: PLINK binary PED files, MACH output format</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://genomecomb.sourceforge.net/">GenomeComb</a></strong>

<ul>
<li>Description: package for analysis of complete genome data; annotation using public data or custom tracks, automated primer desing for Sanger or Sequenom validation; “The cg process_illumina command can be used to generate annotated multisample data starting from fastq files, using tools such as bwa for alignment and GATK and samtools for variant calling. Sequencing data can also be imported from Complete Genomics (cg_process_sample command), Real Time Genomics (cg_process_rtgsample command) and VariantCallFormat (VCF) variant files (vcf2sft command).”</li>
<li>Input: Sequencing data from Complete Genomics, Illumina, SOLiD and VCF;</li>
<li>Output: standard file format used is a simple tab delimited file (.sft, .tsv)</li>
</ul>
</li>
<li>
<strong><a href="http://ancorr.eimb.ru/">Genome Track Analyzer</a></strong>

<ul>
<li>Description: compares genome tracks; allows user to compare DNA expression/binding;</li>
<li>Input: multiple: SGR/TXT, BED, BED6, GFF;  if using prealigned sequence data- use MACS peak caller: BAM, BED, SAM, ELAND</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://animalgene.umn.edu/gvcblub">GVCBLUP</a></strong>

<ul>
<li>Description: animal gene mapping; “genomic prediction and variance component estimation of additive and dominance effects”; standalone program, command line interface, writting in C++ and Java</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://www.jurgott.org/linkage/homog.htm">HOMOG</a></strong>

<ul>
<li>Description: Analyzes heterogeneity with respect to single marker loci or known maps of markers; Carries out homogeneity test for alternative hypothesis “Two family types, one with linkage betweeen a trait to a marker or map of markers, the other without linkage”</li>
<li>Input: HOMOG.DAT - described on website</li>
<li>Output: HOMOG.OUT</li>
</ul>
</li>
<li>
<strong><a href="http://intersnp.meb.uni-bonn.de/">INTERSNP</a></strong>

<ul>
<li>Description: GWIA for case-control SNP and quantitative traits; selected for joint analysis using priori information; Provides linear regression framework, Pathway Association Analysis, Genome-wide Haplotype Analysis,</li>
<li>Input: PLINK input formats (ped/map, tped/tfam, bed/bim/fam) Compatible with SetID files</li>
<li>Gene reference file: Ensembl Release 75</li>
<li>Output: covariance matrix for regression models</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/PMBio/mtSet">mtSet</a></strong>

<ul>
<li>Description: Currently only the standalone version available, but moving to LIMIX software suite; offers set tests- allows for testing between variants and traits; accounts for confounding factors ex. relatedness</li>
<li>Input: sample-to-sample genetic covariance matrix needs to be computed; multiple types of input; simulator requires input genotype and relatedness component;</li>
<li>Output: resdir (result file of analysis), outfile (test statistics and p-values), manhattan_plot (flag)</li>
</ul>
</li>
<li>
<strong><a href="http://dougspeed.com/multiblup/">MultiBLUP</a></strong>

<ul>
<li>Description: Package program, command line interface; constructs linear prediction models; Best Linear Unbiased Prediction; improves upon BLUP involving kinship matrices; options: pre-specified kinships, regional kinships, adaptive multiblups, LD weightings</li>
<li>Input: PLINK format</li>
<li>Output:.reml, .indi.blp</li>
</ul>
</li>
</ol>

<h2>
<a id="variant-annotation" class="anchor" href="#variant-annotation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Variant Annotation</h2>

<ol>
<li>
<strong><a href="http://annovar.openbioinformatics.org/en/latest/">ANNOVAR</a></strong>

<ul>
<li>Description: command-line tool, supports SNPs, INDELs, CNVs and block substitutions, provides wide variety of annotation techniques, depends upon multiple databases (each needing to be downloaded); annotates genetic variants; utilizes RefSeq, UCSC Genes, and the Ensembl gene annotation systems; can compare mutations detected in dpSNP or 1000 Genomes Project; Very popular
*“The final command run TABLE_ANNOVAR, using dbSNP version 138, 1000 Genomes Project 2014 Oct version, NIH-NHLBI 6500 exome database version 2 (referred to as esp6400siv2), dbNFSP version 2.6 (referred to as ljb26), dbSNP version 138 (referred to as snp138) databases and remove all temporary files, and generates the output file called myanno.hg19_multianno.txt”</li>
<li>Input: VCF, ANNOVAR input format (simple text-based format); can convert other formats into ANNOVAR input format</li>
<li>Output: VCF (if input VCF), output file with multiple columns, tab-delimited output file</li>
</ul>
</li>
<li>
<strong><a href="http://wannovar.usc.edu/">wANNOVAR</a></strong>

<ul>
<li>provides web-based access to ANNOVAR software</li>
</ul>
</li>
<li>
<strong><a href="http://genetics.bwh.harvard.edu/pph2/">PolyPhen-2</a></strong>

<ul>
<li>Description: Very popular; Polymorphism Phenotyping; Web application; predicts impact of amino acid substitution on protein; Calculates Bayes posterior probability (Last update July 2015)</li>
<li>Input: FASTA</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://sift.jcvi.org/">SIFT</a></strong>

<ul>
<li>Description: predicts how an amino acid substitution will affect protein function; Based on degree of conservation of amino acid residues- collected though PSI-BLAST; can be applied to nonsynonymous polymorphisms or laboratory-induced missense mutations; links to dbSNP 132, GRCh37; Standalone or web app program; Very popular</li>
<li>Input: Uniprot ID or Accession, Go term ID, Function name, Species Name or ID, etc</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://snpeff.sourceforge.net/">snpEff</a></strong>

<ul>
<li>Description: Genetic variant annotation and effect prediction toolbox; integrated with Galaxy, GATK, and GNKO; can annotate SNPs, INDELs, and multiple-nucleotide polymorphisms; categorizes effects into classes by functionality; Very popular; Standalone or Web app; Claims to calculate all SNPs in 1000 genomes (EMBI) in less than 15 minutes; can annotate SNPs, MNPs, and insertions and deletions; Provides assessment of impact of the variant ( low, medium or high)</li>
<li>Input: VCF, BED</li>
<li>Output: VCF (with new ANN field, also used in ANNOVAR and VEP), HTML summary files</li>
</ul>
</li>
<li>
<strong><a href="http://snpeff.sourceforge.net/SnpSift.html">SnpSIFT</a></strong>

<ul>
<li>Description: Filter and manipulate annotated files; Part of SnpEff main distribution; one variants have been annotated, this can be used to filter your data to find relevant variants</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://www.yandell-lab.org/software/vaast.html">VAAST 2</a></strong>

<ul>
<li>Description: Variant Annotation, Analysis, and Search Tool; probabilistic search tool for identifying damage genes and the disease causing variants; can score both coding and non-coding variants; Four tools: VAT (Variant annotation tool), VST (Variant Selection Tool), VAAST, pVAAST (for pedigree data); updated April 2015</li>
<li>Input: FASTA, GFF3, GVF</li>
<li>Output: CDR (condenser file), VAAST file (both unique to VAAST)</li>
</ul>
</li>
<li>
<strong><a href="http://useast.ensembl.org/info/docs/tools/vep/index.html?redirect=no">VEP</a></strong>

<ul>
<li>Description: (Ensembl) Variant Effect Predictor; determines effect of variants on genes, transcripts, and protein sequence; uses SIFT and PolyPhen</li>
<li>Input: Coordinates of variants and nucleotide changes; whitespace- separated format, VCF, pileup, HGVS</li>
<li>Output: VCF, JSON, Statistics
<hr>
</li>
</ul>
</li>
<li>
<strong><a href="http://www.broadinstitute.org/cancer/cga/absolute">ABSOLUTE</a></strong>

<ul>
<li>Description: (Broad Institute); can estimate purity and ploidy to compute absolute copy number and mutation multiplicitie; reextracts data from the mixed DNA population</li>
<li>Input: HAPSEQ segdat or segmentation file</li>
<li>Output: per-sample output directory and subdirectory providing per-sample text files containing standard out being emitted from R</li>
</ul>
</li>
<li>
<strong><a href="http://www.interactive-biosoftware.com/alamut-batch/">Alamut Batch</a></strong>

<ul>
<li>Description: high-throughput annotation software for NGS analysis; for “intensive variant analysis workflows”; “enriches raw NGS variants with dozens of attributes”; based on clinically oriented Alamut database; Supports human genes; easy to integrate into pipeline (Latest Release- July 2015)</li>
<li>Input:VCF, tab-delimted file</li>
<li>Output: tab-separated file of annotations</li>
</ul>
</li>
<li>
<strong><a href="http://avia.abcc.ncifcrf.gov/apps/site/index">AVIA</a></strong>

<ul>
<li>Description: Annotation, Visualization, and Impact Analysis; “The tool is based on coupling a comprehensive annotation pipeline with a flexible visualization method. We leveraged the ANNOVAR (Wang et. al, 2010) framework for assigning functional impact to genomic variations by extending its list of reference annotation databases (RefSeq, UCSC, SIFT, Polyphen etc.) with additional in-house developed sources (Non-B DB, PolyBrowse).”</li>
<li>Input: BED</li>
<li>Output: Table of annotations with gene annotation features</li>
</ul>
</li>
<li>
<strong><a href="http://bioinformaticstools.mayo.edu/research/bior/">BioR</a></strong>

<ul>
<li>Description: (Mayo Clinic) (Page last updated June 2015) Biological Reference Repository; “data integration tool that enables coordinate based searches and joins based on strings”; “BioR consists of two parts 1) the BioR toolkit which depends on Java…. 2) the BioR catalogs which are the data files used by the system”</li>
<li>Input: VCF</li>
<li>BioR-Supported Catalogs (tar-gzip files): dbSNP, 1000 genomes, HapMap, OMIM, NCBIGene</li>
<li>Output: VCF + JSON</li>
</ul>
</li>
<li>
<strong><a href="http://cadd.gs.washington.edu/">CADD</a></strong>

<ul>
<li>Description: Combined Annotation Dependent Depletion; tool for scoring SNV deletions/insertions; “integrates multiple annotations into one metric”; Score strongly correlates with allelic diversity and pathogenicity; links to 1000 Genome variants; uses Ensembl Variant Effect Predictor</li>
<li>Input: VCF</li>
<li>Output: CADD score</li>
</ul>
</li>
<li>
<strong><a href="http://www2.hu-berlin.de/wikizbnutztier/software/CandiSNPer/">CandiSNPer</a></strong>

<ul>
<li>Description: web application, characterizes SNPs located in vicinity of SNP of interest;</li>
<li>Input: enter SNP ID (rsID), choose population, region, measure for LD, threshold plot format, color of SNPs, and chose to show genes</li>
<li>Output: Imagefile</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/UppsalaGenomeCenter/CanvasDB">CanvasDB</a></strong>

<ul>
<li>Description: “local database infrastructure for analysis of targeted- and whole genome re-sequencing projects”; dependent on MySQL, R, and ANNOVAR</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://www.sanger.ac.uk/resources/software/carol/">CAROL</a></strong>

<ul>
<li>Description: (Wellcome Trust Sanger); Combined Annotation scoRing toOL; Combined functional annotation score of nonsynonymous coding variants; Combines information from PolyPhen-2 and SIFT</li>
<li>Input: tab-delimited with columns obtained from PolyPhen-2 and SIFT output</li>
<li>Output: tab-delimited file</li>
</ul>
</li>
<li>
<strong><a href="http://wiki.chasmsoftware.org/index.php/Main_Page">CHASM</a></strong>

<ul>
<li>Description: Cancer-specific High-throughput Annotation of Somatic Mutations; Last updated May 2014; uses Random Forest Method to “distinguish between driver and passenger somatic mutations”; Positive driver class curated from COSMIC database; packed together with  SNVBox (database)</li>
<li>Input:Passenger mutation rates, Transcript and amino acid change, Genomic coordinates</li>
<li>Output: CHASM score, p-value, FDR</li>
</ul>
</li>
<li>
<strong><a href="http://www.cravat.us/">CRAVAT</a></strong>

<ul>
<li>Description: Cancer-Related Analysis of Variants Toolkit; Web application; Uses CHASM, VEST, SNVGet; “CRAVAT provides predictive scores for germline variants, somatic mutations and relative gene importance, as well as annotations from published literature and databases” Latest Release May 2015;</li>
<li>Input: VCF, CRAVAT format</li>
<li>Output: CRAVAT report- MS Excel spreadsheet or tab-separated file (emailed)</li>
</ul>
</li>
<li>
<strong><a href="http://cupsat.tu-bs.de/">CUPSAT</a></strong>

<ul>
<li>Description: Cologne University Protein Stability Analysis Tool; “tool to predict changes in protein stability upon point mutations”; web service program; Can predict mutant stability from existing PDB structures or custom protein structures</li>
<li>Input:for PDB- provide PDB ID and Amino Acid Residue Number; for custom- PDB file format</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="https://cbcl.ics.uci.edu/public_data/DANN/">DANN</a></strong>

<ul>
<li>Description: Deleterious Annotation of genetic variants; standalone program, uses “the same feature set and training data as CADD to train a deep neural network”; can catch nonlinear relationships; “There are four different datasets: training, validation, testing, and ClinVar_ESP...The ClinVar_ESP dataset is also a testing set containing a set of “gold standard” pathogenic and benign variants”</li>
<li>Input: </li>
<li>Output: </li>
</ul>
</li>
<li>
<strong><a href="http://rulai.cshl.edu/cgi-bin/tools/ESE3/esefinder.cgi?process=matrices">ESEfinder</a></strong>

<ul>
<li>Description: Exonic Splicing Enhancer; useful for interpretation of point mutations/polymorphisms that are disease-associated; GUI interface; web app program</li>
<li>Input: FASTA</li>
<li>Output: html or plain text format, graphical display of results</li>
</ul>
</li>
<li>
<strong><a href="http://www.sanger.ac.uk/resources/software/exomiser/">Exomiser</a></strong>

<ul>
<li>Description: Wellcome Trust Sanger; functionally annotates variants from whole-exome sequencing data; Based on Jannovar and uses UCSC KnownGene; Java program; web app program (Page last modified Feb 2015)</li>
<li>Input: VCF</li>
<li>Output: TSV, VCF</li>
</ul>
</li>
<li>
<strong><a href="https://sites.google.com/site/famannotation/home">FamAn</a></strong>

<ul>
<li>Description: Automated variant annotation pipeline for family-based sequencing studies; Annotaties SNVs and INDELs; 4 models- autosomal dominant, autosomal recessive, de novo mutations and a general model; “A variety of annotations are provided for each segregating variant: number of family (and family ID) each variant hits, variant genomic location and coding effect (based on snpEff), loss-of-function mutation annotation, selected ENCODE annotation, allele frequency in the 1000 Genomes Project, allele frequency in the Exome Variant Server (ESP6500), segmental duplication annotation,  SIFT, PolyPhen2, LRT, MutationTaster, GERP++, PhyloP, SiPhy, etc.” (Last updated May 2014)</li>
<li>Input: VCF</li>
<li>Output: two excel compatible outputs</li>
</ul>
</li>
<li>
<strong><a href="http://www.gene-talk.de/">GeneTalk</a></strong>

<ul>
<li>Description: Combines tool for filtering and data analysis with an online network for genetic professionals; Different degrees- basic license, premium license, in-house solution (the last ones are paid for- Commercial tool?)</li>
<li>Input: VCF</li>
<li>Output: GeneTalk Annotation- includes clinical data, medical relevance, scientific relevance (<a href="http://www.gene-talk.de/public/GeneTalk_Whitepaper_Annotations.pdf">http://www.gene-talk.de/public/GeneTalk_Whitepaper_Annotations.pdf</a>)</li>
</ul>
</li>
<li>
<strong><a href="http://genevetter.kidneyomics.org/">GeneVetter</a></strong>

<ul>
<li>Description: “GeneVetter is a tool designed for investigation of the background prevalence of exonic variation in the Phase 3 1000 Genomes data under user defined filtering criteria”; web app program; GeneVetter uses GRch37p4 (hs37d5.fa.gz), dbSNP build 138, 1000G Phase 3, clinvar_2014072</li>
<li>Input: VCF</li>
<li>Output: TIMS score, summary table, PCA plot</li>
</ul>
</li>
<li>
<strong><a href="http://www.broadinstitute.org/software/cprg/?q=node/31">GSITIC</a></strong>

<ul>
<li>Description: (Broad Institute) Last update- July 2014; Identifies genomic regions that are significantly “amplified or deleted”; Each is given a G score; gives genomic locations and q-values from aberrant regions</li>
<li>Input: segmentation file -seg, markers file -mk (required); -array file list -alf, CNV file -cnv</li>
<li>Reference genome: -refgene (created in MATLAB, GISITIC provides four reference genomes: hg16.mat, hg17.mat, hg18.mat, hg19.mat</li>
<li>Output: All lesions file (text file), amplifications file (text file), deletion genes file (text file), Gistic Scores file, Segmented copy number (pdf file), amplification score GISTIC plot (pdf file), Deletion score/q-vale GISTIC plot (pdf file)</li>
</ul>
</li>
<li>
<strong><a href="http://www.cmbi.ru.nl/hope/about">HOPE</a></strong>

<ul>
<li>Description: Have yOur Protein Explained; Web app program; Automatic mutant analysis server that provides structural effects of a mutation; Uses BLAST against UniProt and PDB along with homology modeling</li>
<li>Input: FASTA protein sequence, or accession code of protein of interest</li>
<li>Output: a report containing information from a “decision tree” and illustrated figures and animations</li>
</ul>
</li>
<li>
<strong><a href="http://umd.be/HSF/">Human Splicing Finder</a></strong>

<ul>
<li>Description: Last update: May 2013; aimed to help study pre-mRNA splicing; combines 12 algorithms to identify mutations’ effect on splicing motifs; uses ensembl database 70</li>
<li>Input: Gene Name, Ensembl transcript ID, Ensembl Gene ID, Consensus CDS, RefSeq Peptide ID, or own sequence (looks like you can enter FASTA)</li>
<li>Output: Chart with columns for predicted signal, predicted algorithm, cDNA position and interpretation</li>
</ul>
</li>
<li>
<strong><a href="http://larva.gersteinlab.org/">LARVA</a></strong>

<ul>
<li>Description: Large-scale Analysis of Variants in noncoding Annotations; New version released July 2015; Command-line program; used for studying noncoding variants; integrates comprehensive set of noncoding elements, modeling their mutation count; Dependent on C++ and BEDtools</li>
<li>Input: multiple</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://www.jurgott.org/linkage/LinkagePC.html">LINKAGE</a></strong>

<ul>
<li>Description:three main programs: mlink (calculates lod scores at fixed values for the recombination fraction in one interval of a genetic map), linkmap (calculates location scores for positions of a disease locus along a marker), and ilink (estimates parameters including recombination fractions, allele frequencies, penetrances, etc)</li>
<li>Input: pedfile (processed by MAKEPED) and datafile (reflects loci for each individual; set in PREPLINK)</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/projects/mnvannotationcorrector/">MAC</a></strong>

<ul>
<li>Description: MNV Annotation Corrector; Ad hoc software, fixes incorrect amino acid predictions that are caused by multiple nucleotide variations; Uses existing annotators ANNOVAR, SnpEff, VEP (last update April 2015) (only 1 download this week → not popular)</li>
<li>Input: List of called SNVs and corresponding BAM</li>
<li>Output: Report identifying block of mutation within codon (BMCs)</li>
</ul>
</li>
<li>
<strong><a href="http://genome.igib.res.in/mitomatic/">mit-o-matic</a></strong>

<ul>
<li>Description: focuses on mtDNA, provides clinically relevant information from different resources; two component pipeline: command link for alignment of NGS reads and online version that provides genetic report on mitocondrial variants</li>
<li>Input:FASTQ, pileup</li>
<li>Reference sequence: rCRSm</li>
<li>Output: Online version gives comprehensive genetic report</li>
</ul>
</li>
<li>
<strong><a href="http://krauthammerlab.med.yale.edu/mutadelic/index.html">Mutadelic</a></strong>

<ul>
<li>Description: Web App program; “This application generates reports on inherited mutations in five genes (ANK1, SLC4A1, SPTA1, SPTB and EPB42) associated with the following rare Mendelian blood disorders: Hereditary Spherocytosis (HS), Hereditary Elliptocytosis (HE) and Hereditary Pyropoikilocytosis”; Newer program- recently validated on omictools</li>
<li>Input: Can upload coordinates of DNA variants or VEP</li>
<li>Output: Displayed on web or can be downloaded in Excel or RDF format</li>
</ul>
</li>
<li>
<strong><a href="http://www.mutationtaster.org/">MutationTaster</a></strong>

<ul>
<li>Description: (Last post on site 2014) Web app program; Rapid evaluation of disease causing alterations; uses NCBI 37 and Ensembl 69</li>
<li>Input: HGNC symbol, NCBI GeneID, or Ensembl ID,</li>
<li>Output: Report containing prediction, summary, name of alteration, etc</li>
</ul>
</li>
<li>
<strong><a href="http://mutpred.mutdb.org/">MutPred</a></strong>

<ul>
<li>Description: web app tool; Classifies amino acids substituation as disease associated or neutral in humans; Last modified Feb. 2014; Based on SIFT, trained using Human Gene Mutation Database</li>
<li>Input:</li>
<li>Output: “The output of MutPred contains a general score (g), i.e., the probability that the amino acid substitution is deleterious/disease-associated, and top 5 property scores (p), where p is the P-value that certain structural and functional properties are impacted.”</li>
</ul>
</li>
<li>
<strong><a href="http://www.broadinstitute.org/cancer/cga/mutsig">MutSigCV</a></strong>

<ul>
<li>Description: (Broad Institute) Mutation Significance (CV= covariates); Analyzes mutations discovered in DNA sequencing to identify genes that were mutated more often than expected</li>
<li>Input: mutations.maf, coverage.txt, covariates.txt</li>
<li>Output: output.txt</li>
</ul>
</li>
<li>
<strong><a href="http://stothard.afns.ualberta.ca/downloads/NGS-SNP/">NGS-SNP</a></strong>

<ul>
<li>Description: Collection of command-line scripts for providing rich SNP annotations; “NCBI, Ensembl, and Uniprot IDs are provided for genes, transcripts and proteins when applicable”;</li>
<li>Input: Samtools consensus pileup, Maq, diBayes, Genetic format, VCF</li>
<li>Output: File containing annotated SNPs is copied from SNP list and some classes are added</li>
</ul>
</li>
<li>
<strong><a href="http://www.broadinstitute.org/oncotator">Oncotator</a></strong>

<ul>
<li>Description: (Broad Institute) “Tool for annotating human genomic point mutations and data relevant to cancer researchers”; Web app; Supports annotation of data from ClinVar, dbSNP, 1000 genomes (plus many other external sites); Only GRCh27 coordinates supported; Last update: April 2015</li>
<li>Input: tal-delimited file</li>
<li>Output: tab-delimited MAF</li>
</ul>
</li>
<li>
<strong><a href="http://omictools.com/panther-s649.html">PANTHER</a></strong>

<ul>
<li>Description: Protein ANalysis THrough Evolutionary Relationships; Web app program, also has its own database; Classification system used to classify proteins and their genes; Also, “Estimates the likelihood of a particular nonsynonymous (amino-acid changing) coding SNP to cause a functional impact on the protein”; Updated in 2015</li>
<li>Input: Data from PANTHER, IDs from Ensembl, EntrezGene, NCBI GI numbers, NCBI UniGene IDs HUGO, UniProt; if ID type is not one of the above, can input txt file or excel format</li>
<li>Output: Analysis results displayed online</li>
</ul>
</li>
<li>
<strong><a href="http://cubio.biology.columbia.edu/pesx/pesx/">PESX</a></strong>

<ul>
<li>Description: Putative Exonic Splicing Enhancers/Silencers; (Can’t tell if this is outdated or not)</li>
<li>Input: FASTA or plain text</li>
<li>Output: Excel spread sheet</li>
</ul>
</li>
<li>
<strong><a href="http://phen-gen.org/index.html">Phen-Gen</a></strong>

<ul>
<li>Description: Combines patient's’ disease symptoms with sequencing data; Standalone or Web app version; Only excepts 1 family per run, in order to evaluate unrelated individuals, each sample needs to be run individually</li>
<li>Input: Variant- VCF; Pheotype- HPO; Pedigree- PED</li>
<li>Output: Combined scores file, variants for top genes file</li>
</ul>
</li>
<li>
<strong><a href="http://mmb.pcb.ub.es/PMut/">PMUT</a></strong>

<ul>
<li>Description: Aimed at annotation and prediction of pathological mutations; based on different kinds of sequence info and neural networks to process information</li>
<li>Input: FASTA</li>
<li>Output; Simple yes/no and reliability index</li>
</ul>
</li>
<li>
<strong><a href="http://provean.jcvi.org/index.php">PROVEAN</a></strong>

<ul>
<li>Description: Protein Variation Effect Analyzer; predicts whether an amino acid substitution or indel has impact on biological function of the protein; “comparable to SIFT or Polyphen-2”; Standalone, Web app, Command line or GUI; Last update May 2014</li>
<li>Input: FASTA, list of variants;</li>
<li>Output: tab-separated columns including Variant, Provean Score and prediciton</li>
</ul>
</li>
<li>
<strong><a href="http://genes.mit.edu/burgelab/rescue-ese/">Rescue-ESE</a></strong>

<ul>
<li>Description: “An online tool for identifying candidate ESEs in vertebrate exons”; Web application; For human, mouse, zebrafish, pufferfish</li>
<li>Input: multi-FASTA or plain text</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://scandb.org/newinterface/index_v1.html">SCAN</a></strong>

<ul>
<li>Description: Web application program, includes a database as well; Database contains physical-based SNP annotations and functional annotations; “Information on physical, functional, and LD annotation served on the SCAN database comes directly from public resources, including the HapMap (release 23a), NCBI (dbSNP 129), or is information created by us using data downloaded from these public resources”; “SCAN can be utilized in several ways including: (i) queries of the SNP and gene databases; (ii) analysis using the attached tools and algorithms; (iii) downloading files with SNP annotation for various GWA platforms”</li>
<li>Input:</li>
<li>Output: HTML, comma-delimited, tab-delimited</li>
</ul>
</li>
<li>
<strong><a href="http://snp.gs.washington.edu/SeattleSeqAnnotation137/">SeattleSeq Annotation</a></strong>

<ul>
<li>Description: “SeattleSeqAnnotation137 was most recently updated October 13, 2013. The current version is 8.08. The most recent site, based on dbSNP build 141, and hg38/NCBI 38”; Provides annotations for SNVs and Indels- includes dbSNP rsID, gene names and accession numbers, variation functions, protein positions and amino acid changes, conservation scores, HapMap frequencies, PolyPhen predictions and clinical association.</li>
<li>Input: Maq, gff, CASAVA, VCF, GATK bed, custom</li>
<li>Output: “default output file format is a header line (starting with "#") followed by tab-separated annotations”; VCF</li>
</ul>
</li>
<li>
<strong><a href="https://cran.r-project.org/web/packages/seqminer/">seqminer 3.7</a></strong>

<ul>
<li>Description: “Efficiently Read Sequence Data (VCF Format, BCF Format and METAL Format) into R”; Command line package program; Published August 2015</li>
<li>Input: VCF, BCF</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="https://genomics.scripps.edu/ADVISER/Home.jsp">SG Adviser</a></strong>

<ul>
<li>Description: Scripps Genome Annotation and Distributed Variant Interpretation Server, web developed applications for variant annotation, “Downstream applications of variant annotation include: Clinical sequencing applications including: carrier testing, or identification of causal variants in molecular diagnosis, tumor sequencing, or diagnostic odyssey. Prioritization of variants prior to statistical analysis of sequence based disease association studies, especially for automated set-generation and enrichment of likely functional variants within sets. Identification of causal variants in post-GWAS/linkage sequencing studies. Identification of causal variants in forward genetic screens (stay tuned for non-human annotation)”</li>
<li>Input: SNV- VCF, BED, and a few others; CNV- BED, CNVator, plus others</li>
<li>Output: tab-delimited file</li>
</ul>
</li>
<li>
<strong><a href="https://rostlab.org/services/snap/">SNAP-2</a></strong>

<ul>
<li>Description: “SNAP2 is a trained classifier that is based on a machine learning device called "neural network". It distinguishes between effect and neutral variants/non-synonymous SNPs by taking a variety of sequence and variant features into account”; predicts impact of amino acid substitution on protein</li>
<li>Input:Protein Sequence in FASTA</li>
<li>Output: “Each model outputs one score for each output class (neutral/effect). These scores of 10 models are averaged in a jury decision. The final score is calculated as the difference of the average score for effect and the average score for neutral’</li>
</ul>
</li>
<li>
<strong><a href="http://snipa.helmholtz-muenchen.de/snipa/">SNiPA</a></strong>

<ul>
<li>Description: “SNiPA offers both functional annotations and linkage disequilibrium information for bi-allelic genomic variants (SNPs and SNVs)”; Based on EMBI 1000 Genomes Project; uses GRCh37 and Ensembl</li>
<li>Input: rs identifiers</li>
<li>Output: list of SNiPA cards</li>
</ul>
</li>
<li>
<strong><a href="http://www.ccmb.med.umich.edu/ccdu/SNPAAMapper">SNPAAMapper</a></strong>

<ul>
<li>Description: “SNPAAMapper is a downstream variant annotation program that can effectively classify variants by region (e.g. exon, intron, etc), predict amino acid change type (e.g. synonymous, non-synonymous mutation, etc), and prioritize mutation effects (e.g. CDS versus 5’UTR, etc).”</li>
<li>Input: VCF in tab-delimited format</li>
<li>Output: spreadsheet result file</li>
</ul>
</li>
<li>
<strong><a href="http://www.tc.umn.edu/%7Ekonox006/Code/SNPMeta/">SNPMeta</a></strong>

<ul>
<li>Description: Generates metadata for SNPs, including gene name, whether the SNP is coding or noncoding, and whether the SNP is synonymous or nonsynonymous; last update October 2013</li>
<li>Input: FASTA</li>
<li>Output: dbSNP submission report format or tab-delimited file</li>
</ul>
</li>
<li>
<strong><a href="http://www.snp-nexus.org/">SNPnexus</a></strong>

<ul>
<li>Description: SNPnexus allows single queries using dbSNP identifiers or chromosomal regions for annotating known variants. The users are also allowed to provide novel in-house SNPs/indels using genomic coordinates on clones, contigs and chromosomes. For practical purposes, SNPnexus allows batch queries comprising SNP data using dbSNP identifiers or genomic coordinates; Web app, also has its own database; Last update May 2014;</li>
<li>Input: genomic position, chromosomal region, dbSNP id, tab-delimited, VCF</li>
<li>Output: Tables for genomic mapping, gene/protein consequences, effect on protein function, hapmap population, regulatory elements, conservation, phenotype and disease association, structural variants</li>
</ul>
</li>
<li>
<strong><a href="http://snps-and-go.biocomp.unibo.it/snps-and-go/">SNPs&amp;Go</a></strong>

<ul>
<li>Description: Predicts human disease-related mutations in proteins with functional annotations; web application; collects framework information;</li>
<li>Input: UNIPROT accession number, mutation postition, wild-type residue, substituting residue</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://figshare.com/articles/Tool_for_rapid_annotation_of_microbial_SNPs_TRAMS_a_simple_program_for_rapid_annotation_of_genomic_variation_in_prokaryotes_/782261">TRAMS</a></strong>

<ul>
<li>Description: Tool for Rapid Annotation of Microbial SNPs; functional annotation of genomic SNPs, annotates as synonymous, nonsynonymous or nonsense; standalone program; have developed workflow in Galaxy</li>
<li>Input: tab-delimited file containing SNP locations, reference nucleotide and SNPs in different strands</li>
<li>Reference genome: Genbank or EMBL format</li>
<li>Output:<br>
</li>
</ul>
</li>
<li>
<strong><a href="http://trinotate.github.io/">Trinotate</a></strong>

<ul>
<li>Description: Transcriptome functional annotation and analysis; “Trinotate makes use of a number of different well referenced methods for functional annotation including homology search to known sequence data (BLAST+/SwissProt/Uniref90), protein domain identification (HMMER/PFAM), protein signal peptide and transmembrane domain prediction (singalP/tmHMM), and comparison to currently curated annotation databases (EMBL Uniprot eggNOG/GO Pathways databases)”</li>
<li>Input: trinity.fasta, BLAST homologies (confusing workflow, don’t really understand, has a ton of dependencies)</li>
<li>Output: Annotation report in .xls</li>
</ul>
</li>
<li>
<strong><a href="http://www.sanger.ac.uk/resources/software/vagrent/">VAGrENT</a></strong>

<ul>
<li>Description: (Wellcome Trust Sanger) Variation Annotation GENeraTor; Suite of perl modules, compares variants with reference genome annotations and generates effects of each variant; Last modified Feb 2014</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://variant.bioinfo.cipf.es/">VARIANT</a></strong>

<ul>
<li>Description: VARIant ANalysis Tool; reports properties of any variant in human, mouse or rat genomes; Requires log in</li>
<li>Input: VCF, GFF, BED</li>
<li>Output: Filtered Variants (same as input); Genes with Variants (txt), Consequence type histogram (txt)</li>
</ul>
</li>
<li>
<strong><a href="http://www.bioconductor.org/packages/2.12/bioc/html/VariantAnnotation.html">VariantAnnotation</a></strong>

<ul>
<li>Description: (Bioconductor) Annotates and filters genetic variants; data is read in VCF and variants are identified according to region (conding, intron, intergenic, spice site, etc); Uses SIFT and PolyPhen to provide protein function predictions; uses 1000 Genomes Chromosome 22 for sample data</li>
<li>Input; VCF</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://bioinformatics.ua.pt/software/variobox/">Variobox</a></strong>

<ul>
<li>Description: Desktop tool for annotation, analysis and comparison of human genes; Standalone program; Annotation data obtained from wave, PDB, or UniProt, LRG, or RefSeq; Researched genes compared to sequences for LRG and RefSeq;</li>
<li>Input: DNA Sequence chromatogram file (.scf, .abi); DNA electropherogram file (.ab1); FASTA</li>
<li>Output: “Gene Panel” (looks like variation viewer)</li>
</ul>
</li>
<li>
<strong><a href="http://genepipe.ncgm.sinica.edu.tw/variowatch/main.do">VarioWatch</a></strong>

<ul>
<li>Description: annotations of genomic variants; Last update February 2015; uses dbSNP 141, annotation release 105;</li>
<li>Input: gene name, SNP ID, position</li>
<li>Output: specially designed views</li>
</ul>
</li>
<li>
<strong><a href="http://vat.gersteinlab.org/">VAT</a></strong>

<ul>
<li>Description: Variant Annotation Lab; Updated May 2014,</li>
<li>Input:VCF, Interval</li>
<li>Output: Gene and sample summaries</li>
</ul>
</li>
<li>
<strong><a href="https://cran.r-project.org/web/packages/WhopGenome/index.html">WhopGenome</a></strong>

<ul>
<li>Description: fast access to whole genome, population scale variation data, Links to UCSC Bioconductor, and AmiGO (for pedigree formats)</li>
<li>Input: VCF, FASTA, Phylip, MAF, plus others</li>
<li>Reference sequence: FASTA</li>
<li>Output:</li>
</ul>
</li>
</ol>

<h2>
<a id="cnv-copy-number-variations-identification" class="anchor" href="#cnv-copy-number-variations-identification" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>CNV (copy number variations) Identification</h2>

<ol>
<li>
<strong><a href="http://bioconductor.org/packages/2.12/bioc/html/cn.mops.html">cn.MOPS</a></strong>

<ul>
<li>Description: Bioconductor; “Mixture of Poissons for CNV detection of NGS data”; models depth of coverage at each genomic position to reduce read count bias; Bayesian approach, written in C++</li>
<li>Input: Read count matrices, genomic range objects (has ability to convert BAM files to these)</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://chipster.csc.fi/manual/cna-define-experiment.html">CNA-Seq</a></strong>

<ul>
<li>Description: Tool in Chipster analytical pipeline, count reads in bins, segment, and call copy number aberrations, plots CNV profiles, identifies common regions, tests DNA copy number induced differential expression, plots combined profiles and copy numbers, adds cytogenic bands, and counts overlapping CNVs by comparing database of genomic variants (<a href="http://dgv.tcag.ca/dgv/app/home">http://dgv.tcag.ca/dgv/app/home</a>)</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="https://code.google.com/p/cnvalidator/">CNValidator</a></strong>

<ul>
<li>Description: Package program, Command line interface; “Identifies CNVs based on homozygous SNPs and the ratio of heterozygous SNP reads”;</li>
<li>Input: text based tab-delimited files</li>
<li>Output: txt output</li>
</ul>
</li>
<li>
<strong><a href="http://compbio.cs.toronto.edu/cnver/">CNVer</a></strong>

<ul>
<li>Description: Package program, command line interface; “supplements the depth-of-coverage with paired-end mapping”; Donor graph- unified framework- reduces sequence biases; only works with human build hg19 and hg18</li>
<li>Input: BAM, tab delimited text</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/abyzovlab/CNVnator">CNVnator</a></strong>

<ul>
<li>Description:Tool for CNV discovery and genotyping </li>
<li>Input: Sam/Bam</li>
<li>Output: CNV calls or genotype results</li>
</ul>
</li>
<li>
<strong><a href="http://www.bioconductor.org/packages/devel/bioc/html/CNVrd2.html">CNVrd2</a></strong>

<ul>
<li>Description: Part of Bioconductor; package program with command line interface; “measure human gene copy number for multiple samples”; written in R</li>
<li>Input: multiple; imports from DNAcopy, IRange, Rsamtools</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/projects/contra-cnv/">CONTRA</a></strong>

<ul>
<li>Description: detects CNVs in whole-exome data, requires Python and R, dependent on BEDtools and SAMtools</li>
<li>Input: BAM/SAM</li>
<li>Reference genome: Ensembl or 1000genomes</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://tvap.genome.wustl.edu/tools/copycat/">CopyCat</a></strong>

<ul>
<li>Description: Currently under “active development” according to website; utilizes multi-core architecture; “detects copy number aberrations by measuring the depth of coverage</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="https://code.google.com/p/gasv/">GASV</a></strong>

<ul>
<li>Description: Geometric Analysis of Structural Variants</li>
<li>Input: BAM</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/projects/asgenseng/?source=directory">GENSENG/AS-GENSENG</a></strong>

<ul>
<li>Description: uses Markhov model; read-depth-based method; identifies regions of discrete copy-number changes while accounting for confounders</li>
<li>Input: BAM</li>
<li>Output: two txt files</li>
</ul>
</li>
<li>
<strong><a href="http://grigoriev.rutgers.edu/software/grom-rd/index.html">GROM-RD</a></strong>

<ul>
<li>Description: Standalone program, command line interface; analyzes multiple biases to detect CNVs. More sensitive than CNVnator or RDXplorer</li>
<li>Input: BAM</li>
<li>Output: “union set of predicted CNV’s from two pipelines”</li>
</ul>
</li>
<li>
<strong><a href="https://www.stt.msu.edu/users/hengwang/mHMM.html">m-HMM</a></strong>

<ul>
<li>Description: written in R, the “mixture hidden Markov model” to detect CNVs, package program, command-line interface</li>
<li>Input:</li>
<li>Output: “data frame of the resulting CNV detection”</li>
</ul>
</li>
<li>
<strong><a href="http://sourceforge.net/projects/magnolya/">Magnolya</a></strong>

<ul>
<li>Description: “de novo CNV detection by co-assembly”; No reference genome needed, compared two NGS datasets; package program, command-line interface, written in Python</li>
<li>Input: FASTA</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://c2s2.yale.edu/software/modSaRa/">modSaRa</a></strong>

<ul>
<li>Description: package program, command-line interface; written in R; “modified Screening and Ranking algorithm”; performs quantile normalization, searches for change-point canidates, eliminates unlikely change points, outputs potential CNV segments</li>
<li>Input:</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://sv.gersteinlab.org/pemer/">PEMer</a></strong>

<ul>
<li>Description: Package program, command-line interface; restricted to academic users only, written in Perl and Python; simulation-based error models; three modules- PEMer workflow, SV-Simulation,   BreakDB (own database);</li>
<li>Input: FASTA</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://www.bioconductor.org/packages/release/bioc/html/QDNAseq.html">QDNAseq</a></strong>

<ul>
<li>Description: (Bioconductor) package program, written in R; WGS method for CNV analysis; dependent on R; “the genome is divided into non-overlapping fixed-sized bins, number of sequence reads in each counted, adjusted with a simultaneous two-dimensional loess correction for sequence mappability and GC content, and filtered to remove spurious regions in the genome”</li>
<li>Input: BAM</li>
<li>Output: “read counts per bin, which have been corrected, filtered, normalized, and optionally log2-transformed”</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/chrisamiller/readDepth">readDepth</a></strong>

<ul>
<li>Description: package program, command-line; written in R; measures depth of coverage obtained by massively parallel sequencing; “niche uses...obsolete on human data”</li>
<li>Input: BED</li>
<li>Output: segs.dat or alts.dat</li>
</ul>
</li>
<li>
<strong><a href="http://www-genepi.med.utah.edu/suppl/SLOPE/slope_guide.txt">SLOPE</a></strong>

<ul>
<li>Description: package program, command-line; written in C++; “detects structural variants from targeted short DNA reads”; quickly detects insertions and deletions</li>
<li>Input:SAM/FASTQ/MAQ</li>
<li>Reference file: FASTA</li>
<li>Output: html file</li>
</ul>
</li>
<li>
<strong><a href="http://bioinformatics.mdanderson.org/main/TIGRA">TIGRA</a></strong>

<ul>
<li>Description: package program, command-line, for academic users only; written in C++; “conducts targeted local assembly of SV”; Uses data from 1000 Genomes</li>
<li>Input: “SV calls and a set of bam files that contain reads mapped to a reference genome such as NCBI build36.”</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://omictools.com/wisecondor-s1816.html">WISECONDOR</a></strong>

<ul>
<li>Description: “WIthin-SamplE COpy Number aberration”; package program, command-line; written in Python; “Detect fetal trisomies and smaller CNV's in a maternal plasma sample using whole-genome data.”</li>
<li>Input: SAM, BAM</li>
<li>Reference file: FASTA</li>
<li>Output: STDOUT plot in PDF</li>
</ul>
</li>
</ol>

<h2>
<a id="sv-identification" class="anchor" href="#sv-identification" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>SV Identification</h2>

<ol>
<li>
<strong><a href="http://gmt.genome.wustl.edu/packages/breakdancer/">BreakDancer</a></strong>

<ul>
<li>Description: Genome wide detection of structural variants; Includes two complementary programs, BreakDancerMini (small indel detection) and BreakDancerMax (5 types of structural variants)</li>
<li>Input: Set of map files produced by a front-end aligner (MAQ, BWA, NovoAlign, and Bfast) and a tab-delimited file containing locations of the map files, detection parameters and sample information</li>
<li>Output:File containing 14 columns</li>
</ul>
</li>
<li>
<strong><a href="https://github.com/tobiasrausch/delly">Delly2</a></strong>

<ul>
<li>Description: Can discover and genotype deletions, tandem duplications, inversions and translocations; uses paired-ends and split-reads; SVs can be visualized using Delly-maze and Delly-sauve</li>
<li>Input: Bam</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://www.broadinstitute.org/software/genomestrip/">Genome STRiP</a></strong>

<ul>
<li>Description: Set of tools for structural variant identification and genotyping; requires genomes from multiple individuals (20-30 minimum); Uses GATK and consists of multiple modules</li>
<li>Input: BAM</li>
<li>Output: VCF</li>
</ul>
</li>
<li>
<strong><a href="http://mrfast.sourceforge.net/">mrFast</a></strong>

<ul>
<li>Description: micro-read Fast Alignment Search Tool; designed to map short reads generate with Illumina</li>
<li>Input:</li>
<li>Output: SAM</li>
</ul>
</li>
<li>
<strong><a href="http://sfu-compbio.github.io/mrsfast/">mrsFast</a></strong>

<ul>
<li>Description: Micro-read substitution-only Fast Alignment Search Tool; designed to map short reads to reference genome assemblies</li>
<li>Input: FastQ
*Reference Genome: FASTA</li>
<li>Output: SAM</li>
</ul>
</li>
<li>
<strong><a href="http://novelseq.sourceforge.net">NovelSeq</a></strong>

<ul>
<li>Description: Designed to detect novel sequence insertions using high-throughput paired-end whole genome sequencing data</li>
<li>Input: FASTA or FASTQ</li>
<li>Output: FASTA file containing paired-end sequences</li>
</ul>
</li>
<li>
<strong><a href="http://sv.gersteinlab.org/pemer/">PEMer</a></strong>

<ul>
<li>Description: Command line package composed of three modules (PEMer workflow, SV-Simulation, and BreakDB) for SV analysis</li>
<li>Input: FASTA</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="http://gmt.genome.wustl.edu/packages/pindel/">Pindel</a></strong>

<ul>
<li>Description: Command line package program for detection of “large deletions, medium sized insertions, inversions, tandem duplications, and other structural variations”</li>
<li>Input: SAM/BAM</li>
<li>Output: pindel raw output format (can be converted to VCF)</li>
</ul>
</li>
<li>
<strong><a href="http://svmerge.sourceforge.net/">SVMerge</a></strong>

<ul>
<li>Description: Pipeline for detecting structural variants by integrating calls from different SV callers</li>
<li>Input: BAM</li>
<li>Output: multiple (depends on SV callers used)</li>
</ul>
</li>
<li>
<strong><a href="http://bioinformatics.mdanderson.org/main/TIGRA">TIGRA</a></strong>

<ul>
<li>Description: Command line package program that “conducts targeted local assembly of structural variants using targeted iterative graph routing assembly algorithm” Uses data from 1000 Genomes project</li>
<li>Input: SV file; group of BAM files</li>
<li>Output: Fasta</li>
</ul>
</li>
</ol>

<h2>
<a id="tool-repositories-analytical-pipelines--workflow-systems" class="anchor" href="#tool-repositories-analytical-pipelines--workflow-systems" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Tool Repositories, Analytical Pipelines, &amp; Workflow Systems</h2>

<h3>
<a id="toolkits" class="anchor" href="#toolkits" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>ToolKits</h3>

<ol>
<li>
<strong><a href="https://github.com/pezmaster31/bamtools">bamtools</a></strong>

<ul>
<li>Description: command line tool kit, also API version; for reading, writing and manipulating BAM (genome alignment) files</li>
<li>Input: BAM</li>
<li>Output: BAM</li>
</ul>
</li>
<li>
<strong><a href="http://www.bioconductor.org/">Bioconductor</a></strong>

<ul>
<li>Description: Open source software project, Popular R module library with over 1000 modules, can analyze diverse high-throughput genomic data; options for multiple different workflows; also available on AMI</li>
<li>Input: Multiple</li>
<li>Output: Multiple</li>
</ul>
</li>
<li>
<strong><a href="https://toolshed.g2.bx.psu.edu/">Galaxy Tool Shed</a></strong>

<ul>
<li>Description: Open, web-based platform; Has 3374 valid tools for many different purposes. Looks like this site has a different tool repository for each different data type, They have agreed to support NCBI genomes, and SRA on a limited basis</li>
<li>Input: Different tools for different types of data</li>
<li>Output: Multiple</li>
</ul>
</li>
<li>
<strong><a href="https://www.broadinstitute.org/gatk/">GATK</a></strong>

<ul>
<li>Description: (Broad Institute) Genome Analysis ToolKit; software library including tools (depth of coverage analyzers, quality score recalibrator, local realigner, SNP/INDEL caller), requires Ant and Java, extensive documentation and wiki system, active community, can also be used to identify somatic mutations</li>
<li>Input: Multiple</li>
<li>Output: Multiple</li>
</ul>
</li>
<li>
<strong><a href="http://broadinstitute.github.io/picard/">picard</a></strong>

<ul>
<li>Description: (Broad Institute) Java based command line tools for manipulating high throughput sequencing data in the BAM format; supported through GATK</li>
<li>Input: BAM, SAM, VCF</li>
<li>Output: BAM, SAM</li>
</ul>
</li>
<li>
<strong><a href="http://samtools.sourceforge.net/">SAMtools</a></strong>

<ul>
<li>Description: provides utilities for manipulating alignments in the SAM format, including sorting, merging, indexing, and generating alignments; can be used for calling SNPs and Indels</li>
<li>Input: Conforms to the specifications produced by GA4GH File Formats working group</li>
<li>Output:</li>
</ul>
</li>
<li>
<strong><a href="https://vcftools.github.io/specs.html">VCFtools</a></strong>

<ul>
<li>Description: set of tools written in Perl and C++ for working with VCF files</li>
<li>Input: VCF, GZVCF, BCF</li>
<li>Output: Multiple, can be piped into another program
)</li>
</ul>
</li>
</ol>

<h3>
<a id="authors-and-contributors" class="anchor" href="#authors-and-contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Authors and Contributors</h3>

<p>Updated July 2016<br>
Created by: Ben Busby and Farrah Hermes<br>
National Center for Biotechnology Information, U.S. National Library of Medicine         </p>

<p><img src="http://getallele.com/images/companies/NCBI.jpg" alt="http://getallele.com/images/companies/NCBI.jpg"></p>

<hr>

<p>An additional thanks to <a href="https://omictools.com/"><strong>OMICtools</strong></a></p>

<p><img src="https://sites.google.com/site/alexandranaba/_/rsrc/1443014124017/in-the-news/thematrisomeprojectreferencedinomictools/omictools-logo.png" alt="https://sites.google.com/site/alexandranaba/_/rsrc/1443014124017/in-the-news/thematrisomeprojectreferencedinomictools/omictools-logo.png"></p>

<hr>

      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/NCBI-Hackathons/Community_Software_Tools_for_NGS">Community software tools for NGS</a> is maintained by <a href="https://github.com/NCBI-Hackathons">NCBI-Hackathons</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
